packages/array-cartesian-product/README.md

```diff
@@ -1,13 +1,20 @@
 ## just-cartesian-product
 
-Part of a [library](../../../../) of zero-dependency npm modules that do just do one thing.
+Part of a [library](https://anguscroll.com/just) of zero-dependency npm modules that do just do one thing.
```

Comment:
Was the change to link to the REPL page instead of github home intentional? I guess that makes sense since the npm page also has a link to github,

Reply:
That url was specified [here](https://github.com/angus-c/just/issues/361#issuecomment-964609500)

Let me know which url you would like

Reply:
My bad. Yeah this is good as is.